containers/hints/anchore_hints.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
we probably want to include:
* binary
* apkg
* dpkg
* rpm

Reply:
maybe nuget can be added too so that when support is added in the platform it can readily use this image